AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/Base Classes/ALHomeViewController.m

```diff
@@ -120,11 +120,16 @@ - (void)attemptSendEmail
     else
     {
         NSString *message = [NSString stringWithFormat: @"Your device is not configured for sending emails.\n\nPlease send emails to %@", kSupportEmail];
-        [[[UIAlertView alloc] initWithTitle: @"Email Unavailable"
-                                    message: message
-                                   delegate: nil
-                          cancelButtonTitle: @"OK"
-                          otherButtonTitles: nil] show];
+        
+        UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Email Unavailable"
+                                                                       message:message
+                                                                preferredStyle:UIAlertControllerStyleAlert];
+        
+        UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault
+                                                              handler:^(UIAlertAction * action) {}];
+        
+        [alert addAction:defaultAction];
+        [self presentViewController:alert animated:YES completion:nil];
```

Comment:
```
        [alert addAction: [UIAlertAction actionWithTitle: @"OK" style: UIAlertActionStyleDefault handler: nil];
        [self presentViewController: alert animated: YES completion: nil];
```

Reply:
Since email functionality is not needed anymore, we are removing this part entirely